lasagne/layers/pool.py

```diff
@@ -217,28 +229,44 @@ def get_output_for(self, input, **kwargs):
 
 
 class FeaturePoolLayer(Layer):
-
     """
-    This layer performs pooling across feature maps, which can be used
-    to implement maxout.
-    IMPORTANT: this layer requires that the number of feature maps is
-    a multiple of the pool size.
+    Feature pooling layer
+
+    This layer pools across a given axis of the input. By default this is axis
+    1, which corresponds to the feature axis for :class:`DenseLayer`,
+    :class:`Conv1DLayer` and :class:`Conv2DLayer`. The layer can be used to
+    implement maxout.
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or tuple
+        The layer feeding into this layer, or the expected input shape.
+
+    pool_size : integer
+        the size of the pooling windows, i.e. the number of features / feature
```

Comment:
Good point. What name is most common in literature? I guess I'm fine standardizing on either "regions" or "windows" (I think "groups" is a bit of a vague term).

The term 'window' is also polyvalent to some extent, so I guess 'region' would be the least confusing. I'll go with that, unless anyone wants to have another name discussion of course ;)